pycrdt_websocket/yroom.py

```diff
@@ -304,3 +306,30 @@ async def serve(self, websocket: Websocket):
                 self.clients.remove(websocket)
         except Exception as exception:
             self._handle_exception(exception)
+
+    def send_server_awareness(self, type: str, changes: tuple[dict[str, Any], Any]) -> None:
+        """
+        Callback to broadcast the server awareness to clients.
+        """
+        if type != "change" or changes[1] != "local":
```

Comment:
Should we actually broadcast every update?
```suggestion
        if type != "update" or changes[1] != "local":
```

Reply:
Your right, we should broadcast all events.
Currently the `update` event is triggered only when the `change` event is triggered.
Should we trigger an update every X sec ?

Reply:
I think we need [this logic](https://github.com/yjs/y-protocols/blob/255b7e6c96d9c6dd48b5519b3f83ab65afdb4bba/awareness.js#L59) in pycrdt. I'll work on it.

Reply:
Yes, that's what I meant.